src/api_server/swagger/firecracker.yaml

```diff
@@ -318,6 +318,29 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /mmds/config:
+    put:
+      summary: Set MMDS configuration.
+      description:
+        Creates a MMDS configuration to be used by the MMDS network stack.
+      parameters:
+        - name: body
+          in: body
+          description: The MMDS configuration as JSON.
+          schema:
+            type: #/definitions/MmdsConfig
```

Comment:
You have to also create the `MmdsConfig` definition. You can find all the definitions at the end of this file.